File: Coding/coding modul 3/PRAK304-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
bilangan_cacah = int(input())

if bilangan_cacah == 0:
    print("Nol")
elif bilangan_cacah >= 1 and bilangan_cacah <= 9:
    print("Satuan")
elif bilangan_cacah == 10:
    print("Puluhan")
elif bilangan_cacah >= 11 and bilangan_cacah <= 19:
    print("Belasan")
elif bilangan_cacah >= 20 and bilangan_cacah <= 99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Coding/coding modul 3/PRAK304-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>

int main()
{
    int bilangan_cacah;

    scanf("%d", &bilangan_cacah);

    if(bilangan_cacah == 0){
        printf("Nol");
    }
    else if(bilangan_cacah >= 1 && bilangan_cacah <= 9){
        printf("Satuan");
    }
    else if(bilangan_cacah == 10){
        printf("Puluhan");
    }
    else if(bilangan_cacah >= 11 && bilangan_cacah <= 19){
        printf("Belasan");
    }
    else if(bilangan_cacah >= 20 && bilangan_cacah <= 99){
        printf("Puluhan");
    }
    else{
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}